staging/src/k8s.io/kubectl/pkg/cmd/apply/apply_test.go

```diff
@@ -788,23 +805,35 @@ func TestApplyPruneObjectsWithAllowlist(t *testing.T) {
 			},
 		},
 	}
-	if err != nil {
-		t.Fatal(err)
-	}
```

Comment:
removed by mistake?

Reply:
This is a copy-paste typo of https://github.com/kubernetes/kubernetes/pull/113116/files#diff-73932b6b01149414cf1793cb2742fb9e07fd8a1de292b49852830273e5d30f96.

The last resource is not applied and no new err is set after the err is already checked.